Resources/Prototypes/_LostParadise/Roadmap/roadmap.yml

```
@@ -0,0 +1,27 @@
+# Короче, думаю тут всё понятно, однако в status 
```

yamllint

[error] trailing spaces

(trailing-spaces)